### DISCUSSION TITLE
Understanding getMarkerFeatures() output

### DISCUSSION THREAD

**Discussion by evaham1** [GreenleafLab/ArchR#1694] [Questions / Documentation] [answered]
Hello and thanks for the great package!

I run the follow code to find all differentially accessible peaks between my clusters:
``
se <- getMarkerFeatures(
  ArchRProj = ArchR, 
  useMatrix = "PeakMatrix", 
  groupBy = "clusters" )
``

I know that I have 178,877 peaks in my ArchR object because I have that many ranges in my ```getPeakSet(ArchR)``` granges object. 
However, the summarised experiment object I get from the code above is a bit smaller, when I run ```dim(rowData(se))``` I see there are only 146,059 rows in this object. 

My understanding of the output of getMarkerFeatures is that each row represents each peak in my ArchR object, and then for each peak there are results from the differential accessibility tests (LogFC, FDRs, etc) for each cluster of cells. I guess that perhaps some peaks are not being outputted to se due to some filters, but I haven't been able to figure out what these filters are and how I can adjust them to make sure my getMarkerFeatures output includes all of my peaks. Any clarification around this would be appreciated!

Many thanks,
Eva

**Comment by rcorces** [comment, marked as answer]
Without a reproducible example, I cannot comment explicitly but one possibility is the filtering performed here:
https://github.com/GreenleafLab/ArchR/blob/f6c0388bd37023400794c9ae8562ad69e3ba9fd7/R/MarkerFeatures.R#L427-L440
